config/sync/views.view.stanford_events.yml

```diff
@@ -1126,7 +1126,7 @@ display:
             trim: 0
             preserve_tags: ''
             html: 0
-          element_type: h3
+          element_type: h2
```

Comment:
All views should have h3 headings since the list paragraph requires a headline field that is h2.

Is there a reason for this change?

Reply:
Just noticed that on this page, it goes from h1 to h3: https://amptesting.sites.stanford.edu/events
But I figured in retrospect it was due to the list paragraph containing the Headline field. I'll revert this heading change in the view. 😃